test/common/ssl/utility_test.cc

```diff
@@ -52,12 +52,12 @@ TEST(UtilityTest, TestGetSerialNumber) {
 
 TEST(UtilityTest, TestDaysUntilExpiration) {
   bssl::UniquePtr<X509> cert = readCertFromFile("test/common/ssl/test_data/san_dns_cert.pem");
-  EXPECT_EQ(270, Utility::getDaysUntilExpiration(cert.get()));
+  EXPECT_LE(0, Utility::getDaysUntilExpiration(cert.get()));
```

Comment:
Can we inject TimeSource into getDaysUntilExpiration?

Reply:
Yeah, +1, if we can do this quickly and unblock CI, this is the right thing to do. If we need something to just unblock CI, lets do the current PR with the TODO.

Reply:
Yes I left it as a TODO, didn't want to introduce more to unblock CI quickly.